x-pack/plugins/threat_intelligence/server/utils/get_runtime_mappings.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import { threatIndicatorNamesOriginScript, threatIndicatorNamesScript } from './display_name';
+import { threatIndicatorNamesOriginScript, threatIndicatorNamesScript } from './indicator_name';
```

Comment:
to be consistent with the rest of the plugin (after clean up) we could add a `index.ts` at the root of this `utils` folder and clean up the imports